src/redux/auth/reducer.js

```diff
@@ -0,0 +1,23 @@
+import * as actionTypes from "./actionTypes";
+
+const authStoreInitialState = {
+  isSignIn: false
+};
+
+const authReducer = (state = authStoreInitialState, action) => {
+  switch (action.type) {
+    case actionTypes.SIGN_IN_AND_SET_JWT_SUCCESS: {
+      return { isSignIn: true };
+    }
+    case actionTypes.SIGN_OUT_AND_REMOVE_JWT_SUCCESS: {
+      return { isSignIn: false };
+    }
+    case actionTypes.JWT_VALIDATE_TURE: {
+      return { isSignIn: false };
+    }
+    default:
+      return state;
+  }
+};
+
```

Comment:
```suggestion
import {
SIGN_IN_AND_SET_JWT_SUCCESS,
SIGN_OUT_AND_REMOVE_JWT_SUCCESS,
JWT_VALIDATE_TURE,
} from "./actionTypes";

const authStoreInitialState = {
  isSignIn: false
};

const authReducer = (state = authStoreInitialState, { type }) => {
  switch (type) {
    case SIGN_IN_AND_SET_JWT_SUCCESS: {
      return { isSignIn: true };
    }
    case SIGN_OUT_AND_REMOVE_JWT_SUCCESS: {
      return { isSignIn: false };
    }
    case JWT_VALIDATE_TURE: {
      return { isSignIn: false };
    }
    default:
      return state;
  }
};
```

ㅎㅎ es6의 디스트럭처링을 사용하는 건 어떠시나유

Reply:
코드가 더 간결해 보이네요 👍

Reply:
근데 action이 추가 됐을 경우를 대비해서 *을 이용해 import하는 것도 좋지 않을까요??